examples/chip-tool/BUILD.gn

```diff
@@ -32,7 +32,12 @@ config("config") {
     "${chip_root}/src/lib",
   ]
 
-  defines = [ "CONFIG_USE_SEPARATE_EVENTLOOP=${config_use_separate_eventloop}" ]
+  default_paa_trust_store_path = rebase_path(chip_root, "")
+
+  defines = [
+    "CONFIG_USE_SEPARATE_EVENTLOOP=${config_use_separate_eventloop}",
+    "DEFAULT_PAA_TRUST_STORE_PATH=\"${default_paa_trust_store_path}/credentials/development/paa-root-certs\"",
+  ]
```

Comment:
This prevents porting the chip-tool and/or cross-compiling the chip-tool on a better host and then move it to a target

Reply:
That's a basic issue with requiring use of a file-based trust store, right?